.aspect/rules/external_repository_action_cache/npm_translate_lock_LTYxNjU0OTY4OQ==

```diff
@@ -0,0 +1,26 @@
+# @generated
```

Comment:
Why was this added?

Reply:
The label format that went into the hash generation changed. Fixed it up.